Tests/UtilityTests/ProcessTests.swift

```diff
@@ -99,11 +113,9 @@ class ProcessTests: XCTestCase {
             }
             XCTAssertEqual(process.processID, ProcessID(parent))
             // We should have killed the process and any subprocess spawned by it.
-            XCTAssertFalse(running(ProcessID(parent)))
-            // This returns true sometimes: https://bugs.swift.org/browse/SR-3538
-          #if false
-            XCTAssertFalse(running(ProcessID(child)))
-          #endif
+            XCTAssertFalse(try running(ProcessID(parent)))
+            // The child process becomes defunct when executing the tests using docker directly without entering bash.
```

Comment:
Might make sense to mark this as a `FIXME` to eventually investigate why that might be the case and/or if it is something we should maybe handle/workaround in `Process`.